.gitignore

```diff
@@ -48,3 +49,10 @@ atlassian-ide-plugin.xml
 
 # Plugins
 /providedLib
+
+# Gradle
+.gradle
+
+# Other files
+/local.properties
+*.jar
```

Comment:
add
`!gradle/wrapper/gradle-wrapper.jar`
because it is needed if you will use a wrapper